### PR TITLE
pythonPackages.cloudflare: init at 2.6.5

### PR DESCRIPTION
**Pull request by costrouc**
###### Motivation for this change

Using this for my own development and wanted to push it upstream.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by costrouc** [comment]
@GrahamcOfBorg build python2Packages.cloudflare python3Packages.cloudflare python2Packages.auth0-python python3Packages.auth0-python

**Comment by privateseabass** [comment]
This worked for me, the bin showed accurate version.

File: pkgs/development/python-modules/cloudflare/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    pyyaml
+    jsonlines
+  ];
+
+  # no tests associated with package
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python wrapper for the Cloudflare v4 API";
+    homepage = https://github.com/cloudflare/python-cloudflare;
```
Comment (jonringer) [inline pkgs/development/python-modules/cloudflare/default.nix:31]: rfc 45
```suggestion
    homepage = "https://github.com/cloudflare/python-cloudflare";
```

File: pkgs/development/python-modules/cloudflare/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  propagatedBuildInputs = [
+    requests
+    future
+    pyyaml
+    jsonlines
+  ];
+
+  # no tests associated with package
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/python-modules/cloudflare/default.nix:27]: ensure it's not trivially broken
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "CloudFlare" ];
```

File: pkgs/development/python-modules/cloudflare/default.nix (hunk 1 above)
Comment (costrouc) [reply pkgs/development/python-modules/cloudflare/default.nix]: Fixed.

File: pkgs/development/python-modules/cloudflare/default.nix (hunk 2 above)
Comment (costrouc) [reply pkgs/development/python-modules/cloudflare/default.nix:27]: Fixed.

**Comment by costrouc** [comment]
Corrections made.

@GrahamcOfBorg build python2Packages.cloudflare python3Packages.cloudflare python2Packages.auth0-python python3Packages.auth0-python

**Comment by costrouc** [comment]
^ what would have caused this to fail? I only put the urls in quotes since

**Comment by jonringer** [comment]
borg was broken on master, rebase the PR and it should fix it:
```
git pull -r origin/master
git push .. .. --force
```

**Comment by costrouc** [comment]
@GrahamcOfBorg build python2Packages.cloudflare python3Packages.cloudflare python2Packages.auth0-python python3Packages.auth0-python

**Comment by costrouc** [comment]
Ready for merge

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 85773 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>6 packages built:</summary><br>- python27Packages.auth0-python<br>- python27Packages.cloudflare<br>- python37Packages.auth0-python<br>- python37Packages.cloudflare<br>- python38Packages.auth0-python<br>- python38Packages.cloudflare</details>

**Pull request merged**